checks/frozen_deps.go

```diff
@@ -28,10 +31,76 @@ func init() {
 
 // FrozenDeps will check the repository if it contains frozen dependecies.
 func FrozenDeps(c *checker.CheckRequest) checker.CheckResult {
+	return checker.MultiCheckAnd(
+		lockFilesPresent,
+		workflowActionsPinned,
+	)(c)
+}
+
+// TODO: need to support Docker https://github.com/ossf/scorecard/issues/403
+
+// Check file content
+func onFileContent(path string, content []byte,
+	logf func(s string, f ...interface{})) bool {
+
+	// Structure for workflow config.
+	// We only retrieve what we need for logging.
+	// Github workflows format: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+	type YamlWorflowConfig struct {
+		Name string `yaml:name`
+		Jobs map[string]struct {
+			Name  string `yaml:name`
+			Steps []struct {
+				Name string `yaml:name`
+				Id   string `yaml:id`
+				Uses string `yaml:uses`
+			}
+		}
+	}
+
+	var workflow YamlWorflowConfig
+	err := yaml.Unmarshal(content, &workflow)
+	if err != nil {
+		panic(fmt.Errorf("!! frozen-deps - Cannot unmarshall file %v\n%v\n%v", path, content, string(content)))
```

Comment:
s/Cannot unmarshall/cannot unmarshal ?
also content is repeated twice, maybe not good to show content there, just file path is enough. otherwise usually unreable messag.e